src/Repository/CompanyRepository.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Company;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+
+/**
+ * @method Company|null find($id, $lockMode = null, $lockVersion = null)
+ * @method Company|null findOneBy(array $criteria, array $orderBy = null)
+ * @method Company[]    findAll()
+ * @method Company[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
```

Comment:
I don't think we should maintain this kind of doc block. Pretty decent IDEs can handle with the class' methods.

Reply:
done